packages/widgets/src/dockpanel.ts

```diff
@@ -347,11 +347,11 @@ class DockPanel extends Widget {
    *
    * @param msg - The message sent to the widget.
    */
-  processMessage(msg: Message): void {
+  processMessage(msg: Message): void | Promise<void> {
     if (msg.type === 'layout-modified') {
       this._layoutModified.emit(undefined);
```

Comment:
Question:  Does this effectively "eat" the layout-modified message?  I wasn't 100% sure if it should return a Promise at this pont?